pex/variables.py

```diff
@@ -189,8 +189,8 @@ def from_rc(cls, rc=None):
         """
         ret_vars = {}  # type: Dict[str, str]
         rc_locations = [
-            "/etc/pexrc",
-            "~/.pexrc",
+            os.path.join(os.sep, "etc", "pexrc"),
```

Comment:
Does the path `/etc/pexrc` make sense on OSes where path sep is not `/` ?

Reply:
Dunno, but no harm if not.